libraries/launcher/org/prismlauncher/log4jinjector/UniversalLog4JInjector.java

```diff
@@ -0,0 +1,41 @@
+package org.prismlauncher.log4jinjector;
+
+import java.util.logging.Logger;
+import java.util.logging.Level;
+
+public class UniversalLog4JInjector {
+    private static final Logger LOGGER = Logger.getLogger("UniversalLog4JInjector");
```

Comment:
In #291, java.util.logging was replaced with a custom Log class.

Reply:
New commit uses the custom Log